3.Tests/UI/StepDefinitions/Settings/Profile/UpdateFullNameStepDefinition.cs

```diff
@@ -0,0 +1,69 @@
+﻿using OpenQA.Selenium;
+using SeleniumTest.Core.Drivers;
+using SeleniumTest.Core.Interfaces;
+using SeleniumTest.Tests.Steps.Commons;
+using TechTalk.SpecFlow;
+using Views.WebAppPages;
+
+namespace SeleniumTest.Tests;
+
+[Binding]
+[Scope(Feature = "Full Name update")]
+[TestFixture]
+public class UpdateFullNameStepDefinition : CommonSteps
+{
+    private readonly ProfilePage _profile;
+    private readonly IGenericWebDriver _driver;
+    private readonly ScenarioContext _scenarioContext;
+
+    public UpdateFullNameStepDefinition(ScenarioContext scenarioContext, ChromeWebDriver driver) : base(scenarioContext, driver)
+    {
+        _scenarioContext = scenarioContext;
+        _driver = driver;
+        _profile = new ProfilePage(driver);
+    }
+
+    [Given(@"the user clicks on the Settings option on the Nav Bar")]
+    public void GiventheuserclicksontheSettingsoptionontheNavBar()
+    {
+        _profile.Settings.Click();
+        IWebElement settingsDialog = _driver.Instance().FindElement(By.XPath("//div[@id='settingsDialog']"));
+        Assert.True(settingsDialog.Displayed);
+    }
+
+    [When(@"the user clicks on the Full Name input")]
+    public void WhentheuserclicksontheFullNameinput()
+    {
+        _profile.FullNameTextField.Clear();
```

Comment:
The step definition says the user is clicking on an input, but this implementation clears the input, step definition should match what the step implementation does

Reply:
Yes, I thought the same when I was doing it. It would be okay if I add a method "Click()" to the ITypeable even if it's not necessary to send the keys? Because I added this step on the feature file because in a real scenario the user would click on the input.

Reply:
Fixed. I refactored the feature file and now I have a step for introducing the new name directly